test/e2e/log/log.go

```diff
@@ -0,0 +1,54 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 Nutanix
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+const (
+	LogDebug = "DEBUG"
+	LogInfo  = "INFO"
+	LogWarn  = "WARN"
+	LogError = "ERROR"
+)
+
+func Debugf(format string, a ...interface{}) {
+	Logf(LogDebug, format, a...)
+}
+
+func Infof(format string, a ...interface{}) {
+	Logf(LogInfo, format, a...)
+}
+
+func Warnf(format string, a ...interface{}) {
+	Logf(LogWarn, format, a...)
+}
+
+func Errorf(format string, a ...interface{}) {
+	Logf(LogError, format, a...)
+}
+
+func Logf(level string, format string, a ...interface{}) {
+	msg := level + ": " + format + "\n"
+	fmt.Fprintf(GinkgoWriter, msg, a...)
+}
```

Comment:
Add a comment above all exported funcs and consts.

Reply:
Done